pkg/cmd/get/get_build_logs.go

```diff
@@ -505,27 +521,26 @@ func (o *GetBuildLogsOptions) loadBuilds(kubeClient kubernetes.Interface, ns str
 	}
 	builds.SortBuildPodInfos(buildInfos)
 	if len(buildInfos) == 0 {
-		return names, defaultName, buildMap, pipelineMap, fmt.Errorf("no knative builds have been triggered which match the current filter")
+		return names, defaultName, buildMap, fmt.Errorf("no knative builds have been triggered which match the current filter")
 	}
 
 	for _, build := range buildInfos {
 		name := build.Pipeline + " #" + build.Build
 		names = append(names, name)
-		buildMap[name] = build
-		pipelineMap[build.Pipeline] = build
+		buildMap[name] = append(buildMap[name], build)
 
 		if build.Branch == "master" {
 			defaultName = name
 		}
 	}
-	return names, defaultName, buildMap, pipelineMap, nil
+	return names, defaultName, buildMap, nil
 }
 
-func (o *GetBuildLogsOptions) loadPipelines(kubeClient kubernetes.Interface, tektonClient tektonclient.Interface, jxClient versioned.Interface, ns string) ([]string, string, map[string]builds.BaseBuildInfo, map[string]builds.BaseBuildInfo, error) {
+// LoadPipelines loads all available pipelines as PipelineRunInfos.
+func (o *GetBuildLogsOptions) LoadPipelines(kubeClient kubernetes.Interface, tektonClient tektonclient.Interface, jxClient versioned.Interface, ns string) ([]string, string, map[string][]builds.BaseBuildInfo, error) {
```

Comment:
Why do you need to export this function? Is it just for testing? If so, then I think this is an anti-pattern. Then I rather put the test into the same package.